apps/extension/src/components/Request/index.tsx

```diff
@@ -0,0 +1,160 @@
+import { View, Image, Text, TouchableOpacity } from '@walless/ui';
+import { FC } from 'react';
+import styles from './style';
```

Comment:
There is no `./style` file, and we don't use/import `css` at component level, in fact don't use `css` directly.